bin/selenium-standalone

```diff
@@ -3,41 +3,74 @@
 var minimist = require('minimist');
 var which = require('which');
 var merge = require('lodash').merge;
+var path = require('path');
 
 var selenium = require('../');
 var defaultConfig = require('../lib/default-config');
 
-which('java', function javaFound(err, javaPath) {
-  if (err) {
-    console.error('Could not find `java`, make sure it is installed in your $PATH');
-    return;
-  }
+function isAbsolutePosix(path) {
+  return path.charAt(0) === '/';
+}
+
+function isAbsoluteWin32(path) {
+  // https://github.com/nodejs/node/blob/b3fcc245fb25539909ef1d5eaa01dbf92e168633/lib/path.js#L56
+  var splitDeviceRe = /^([a-zA-Z]:|[\\\/]{2}[^\\\/]+[\\\/]+[^\\\/]+)?([\\\/])?([\s\S]*?)$/;
+  var result = splitDeviceRe.exec(path);
+  var device = result[1] || '';
+  var isUnc = Boolean(device && device.charAt(1) !== ':');
+
+  // UNC paths are always absolute
+  return Boolean(result[2] || isUnc);
+}
 
+function isAbsolute(path) {
+  return (process.platform === 'win32') ? isAbsoluteWin32(path) : isAbsolutePosix(path);
+}
+
+function parseCommandAndOptions(javaPath) {
   var argv = minimist(process.argv.slice(2), {
     string: ['version', 'drivers.chrome.version', 'drivers.ie.version', 'drivers.firefox.version']
   });
 
   var action = argv._[0];
 
+  if (!action) {
+    throw new Error('No action provided');
+  }
   if (action !== 'install' && action !== 'start') {
-    console.log('Usage: selenium-standalone install or selenium-standalone start');
-    return;
+    throw new Error('Invalid action "'+ action + ' (Valid actions are [' + Object.keys(actions).join(', ') + '])');
   }
 
   // everything after `selenium-standalone install [options] --` will be in argv._
   var seleniumArgs = argv._.slice(1);
 
-  // build a new map removing `_` from argv
+  // build a new map removing `_` and --config from argv
   var options = Object.keys(argv).reduce(function(prev, cur) {
-    if (cur !== '_') {
+    if ((cur !== '_') && (cur !== 'config')) {
       prev[cur] = argv[cur];
     }
 
     return prev;
   }, {});
 
-  // Merge in default options
-  options = merge({}, defaultConfig, options);
+  // If a config file was specified, load it
+  var configFromFile = {};
+
+  if (argv.config) {
+    try {
+      configFromFile = require(
+        isAbsolute(argv.config) ? argv.config : path.resolve(process.cwd(), argv.config)
```

Comment:
It's only a workaround for the UNC bug on paths on windows (`\\host\path`)
```
C:\Users\jcrouzet>node -v
v6.9.2

C:\Users\jcrouzet>node
>  path.resolve(process.cwd(), 'c:\\site\\config')
'c:\\site\\config'
> path.resolve(process.cwd(), '\\\\nfs-share\\config')
'C:\\nfs-share\\config'
>
```

I agree it's a very rare case (just been traumatized by this one in past, sort of pavlov reflex).
I can remove it if it seems over-engineered